robot_sort/robot_sort.py

```diff
@@ -97,6 +97,24 @@ def sort(self):
         Sort the robot's list.
         """
         # Fill this out
```

Comment:
Nice implementation. All tests are passing. It was great working with you this week Derek. Keep the hard work and see you next week.